chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -eo pipefail
+# Args check
+if [[ "$#" -ne 2 ]]; then
+    echo "Error: Two arguments are required"
+    exit 1
+fi
+# Function for delimiter
+print_separator() {
+    printf '%*s\n' "${COLUMNS:-$(tput cols)}" '' | tr ' ' -
+}
+# Function to generate a random string and write to a file with permissions
+gen_rand_str() {
```

Comment:
You function name may be incorrectly understood, i.e. you write that you just create random string `get_rand_str`, but inside you create random string, push it to a file and set some permissions to file. Please always name functions in the way to represent what they do.

```suggestion
create_file() {
```